Preference Access SSN Matching/REPWRITERSPECS/RB.PREFACCESS.SSNMATCHING

```diff
@@ -283,7 +283,7 @@ SETUP
   CALL BNOSETPARAMVALS
  ELSE
   DO
-   PRINT "Error reading CFG Letter file "+BNODEBUGLFTITLE+FORMAT(": 999",BNOERRORCODE)+
+   PRINT "Error reading CFG Letter file "+BNOCFGFILENAME+FORMAT(": 999",BNOERRORCODE)+
          " - "+BNOCFGFILEFERROR
    NEWLINE
    TERMINATE
```

Comment:
Line 729 has "COL=001 FORMAT("R999999",ACCOUNTCOUNT)"
Does the R do something in Format?

Reply:
The "R" was originally referring to the "record" number but I've sense added that reference to the header line.  Removed